app/ui/src/app/integrations/list-page/list-page.component.ts

```diff
@@ -62,17 +63,22 @@ export class IntegrationsListPage implements OnInit {
       status: number,
       headers: ParsedResponseHeaders
     ) => {
-      if (status === 204) {
-        this.notificationService.popNotification({
-          type: NotificationType.SUCCESS,
-          header: 'Imported!',
-          message: 'Your integration has been imported'
-        });
+      if (status === 200) {
+        const changeEvents = JSON.parse(response) as ChangeEvent[];
+        for ( const x of changeEvents ) {
```

Comment:
Unwanted space inside parens. We definitely need to fix the TypeScript Linter. Sorry for that.

Reply:
Please fix linter if these types of things bother you that much.  Just commenting on this is consuming time on things that do not add value to the finished product.